Sofa/framework/Helper/src/sofa/helper/system/PluginManager.cpp

```diff
@@ -199,17 +199,21 @@ PluginManager::PluginLoadStatus PluginManager::loadPluginByPath(const std::strin
             if (errlog) (*errlog) << msg << std::endl;
             return PluginLoadStatus::MISSING_SYMBOL;
         }
-        getPluginEntry(p.getModuleName,d);
+
+        if(!getPluginEntry(p.getModuleName,d))
+        {
+            dmsg_warning("PluginManager") << DynamicLibrary::getLastError();
```

Comment:
I am not sure the triggered log message is clear here for a newbie. I am talking about `is different from where it was supposed to be loaded`. Can you try to improve it?

And why do you add a warning here and not for `getModuleDescription`, `getModuleLicense`, ... ?

Reply:
> I am not sure the triggered log message is clear here for a newbie. I am talking about `is different from where it was supposed to be loaded`. Can you try to improve it?
> 
I guess I can 🫠

> And why do you add a warning here and not for `getModuleDescription`, `getModuleLicense`, ... ?

Because moduleName is "important" because it is used as "plugin name" and is used by PluginManager::getPluginByName(), pluginIsLoaded() among others. 
Description, license, etc is only used by GUIs.

Actually this is something I was wondering while doing this PR, should we request the get*() to be mandatorily implemented ?